shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -54,7 +54,7 @@
   private Context context;
 
   // The View currently rendering the Flutter UI associated with these platform views.
-  private View flutterView;
+  private FlutterView flutterView;
```

Comment:
Does `PlatformViewsController` still need to support the legacy `io.flutter.view.FlutterView`?

Reply:
I don't believe so since we migrated to the new embedding

Reply:
Actually, I will change that back and investigate if the old embedding isn't supported.

Reply:
Done.

Reply:
@xster do you know if the old Android embedding is not longer supported?